src/main/java/frc/core/util/pid/TalonVelocity.java

```diff
@@ -75,21 +76,26 @@ public TalonVelocity(
       followers
     );
   }
-  
-  private void setVelocity(double velocity, double dutyCycle) {
-    this.velocityUnitsPer100ms = dutyCycle * velocity * 4096 / 600;
+
+  private void setVelocity(double rpm, double dutyCycle) {
+    dutyCycle = rpm > (ShooterConstants.maxRPM * dutyCycle) ? dutyCycle : 1;
```

Comment:
bruno, acho que o rpm não precisa ser maior que 80% dp rpm máximo pra usar o dutyCycle, acho que direito quando o rpm for maior que o rpm máximo, é quando deve utilizar o dutyCycle